### PR TITLE
display options at the right and always available

### PR DESCRIPTION
**Pull request by Johann-S**
I change how we display our options, I hope you'll like it 😄 

/CC @MartijnCuppens , @XhmikosR

### CODE REVIEW

**Comment by XhmikosR** [comment]
Not sure I like it personally.

Let alone that this doesn't scale nice. I'd just leave things as they were.

**Comment by Johann-S** [comment]
Hmm I have to check on large screen, but on my 13" screen it looks nice 🤔

**Comment by XhmikosR** [comment]
It just doesn't add any value IMO, UI-wise it's way worse. No need to have it fixed either. Just use the grid system.

**Comment by Johann-S** [comment]
I added a bit of responsive 👍

File: src/main.css [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+.col-options {
+  right: 0;
+}
+
+@media screen and (min-width: 200px) and (max-width: 640px) {
+  .col-options {
+    position: relative !important;
+  }
+
+  .col-options.col-3 {
```
Comment (MartijnCuppens) [inline src/main.css]: Definitely not a fan of this. Use the responsive grid classes for this.

Comment (Johann-S) [reply src/main.css]: Yep but I used automatics `col` and to archieve that I should add/remove classes 😟 If you can help me on that do not hesitate 👍

Comment (MartijnCuppens) [reply src/main.css]: I'll have a look at it this weekend

**Comment by Johann-S** [comment]
Ok so I'll close my PR

Maybe we can add a Scrollspy on the right ?

**Pull request closed without merging**